### ISSUE TITLE
Escape hatch / disable rewrites over regions

### ISSUE THREAD

**Issue by olafurpg** [scalacenter/scalafix#241]
Currently, scalafix runs all rewrites with no option to opt-out of rewrites over particular regions. It would be nice if scalafix offered an "escape hatch" to disable rewrites for particular regions. It could support a syntax like described by @ShaneDelmore in #238 (inlined below)

```
//scalafix: off. –disables scalafix for the rest of the file until
//scalafix: on is found
//scalafix: ignore –don’t scalafix this line (or method?) at all
//scalafix: ignore.RemoveUnused –Only ignore the RemoveUnused rewrite but apply any other matching rewrites.
//scalafix: ignore.{ RemoveUnused, ExplicitReturnTypes } –handle multiple ignores with a syntax similar to imports maybe?
```

**Comment by olafurpg** [comment]
Related discussion in https://github.com/scalacenter/scalafix/issues/238#issuecomment-312446114

**Comment by olafurpg** [comment]
Escape hatches for rewrites are blocked by #368. This should be quite simple to implement for linters, since each linter message is attached to a single position.

**Comment by olafurpg** [comment]
Fixed by @MasseGuillaume in #415
